client/components/tinymce/plugins/mentions/rest-client/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Module dependencies.
+ */
+import EventEmitter from 'events/';
+import wpcom from 'lib/wp';
+
+const debug = require( 'debug' )( 'calypso:tinymce:mentions' );
+
+export default class Client extends EventEmitter {
+	constructor() {
+		super();
+	}
+
+	getUsersSuggestions( siteId ) {
+		const key = 'suggestions-' + siteId;
+
+		if ( ! this.suggestions ) {
+			this.suggestions = {};
+		}
+
+		if ( this.suggestions[ key ] ) {
+			return;
+		}
+
+		this.suggestions[ key ] = [];
+		this.emit( 'loading' );
+
+		try {
+			if ( null != sessionStorage.getItem( key ) ) {
+				const newSuggestions = JSON.parse( sessionStorage.getItem( key ) );
+
+				this.suggestions[ key ] = newSuggestions;
+				this.emit( 'loadingDone', this.suggestions[ key ] );
+
+				return;
+			}
+		} catch ( e ) {
+			debug( "couldn't get sessionStorage item for: %s", key );
+		}
+
+		wpcom.undocumented().userSuggestions().get(
```

Comment:
There is a wpcom-undocumented helper lib to place these: https://github.com/Automattic/wp-calypso/tree/master/client/lib/wpcom-undocumented